packages/logger/src/logger.injectable.ts

```diff
@@ -56,24 +63,23 @@ export const logSillyInjectionToken = getInjectionToken<LogFunction>({
 
 const screamingKebabCase = (str: string) => pipeline(str, kebabCase, toUpper);
 
-const getLogFunctionFor =
-  (scenario: keyof Logger) =>
-  (di: DiContainerForInjection): LogFunction => {
+const getLogFunctionFor = (scenario: keyof Logger, namespace: string | undefined) => {
+  const prefix = namespace
+    ? `[${screamingKebabCase(namespace.replace(/-feature$/, ""))}]: `
```

Comment:
we assume the id or namespace is lowercase?

Reply:
If somebody called their feature "my-feature"?

Reply:
Yeah I did assume that given that all of them are. I can make a follow-up PR to fix this